src/main/webapp/app/core/course/manage/user-management-dropdown/user-management-dropdown.component.html

```diff
@@ -0,0 +1,16 @@
+<div ngbDropdown class="d-inline">
+    <button id="user-management-dropdown" class="btn btn-secondary" ngbDropdownToggle>
+        <fa-icon [icon]="faUser" />
+        <span class="d-none d-md-inline"> {{ 'global.menu.admin.userManagement' | artemisTranslate }}</span>
+    </button>
+    <div ngbDropdownMenu aria-labelledby="instructor-actions-dropdown">
+        @for (userAddAction of userAddActions; track userAddAction) {
+            <a class="btn" [routerLink]="userAddAction.routerLink" ngbDropdownItem [id]="userAddAction.id">
+                @if (userAddAction.icon) {
+                    <fa-icon [fixedWidth]="true" [icon]="userAddAction.icon" class="ms-2 me-3" />
+                }
+                <span [jhiTranslate]="userAddAction.translationKey"></span>
```

Comment:
`[jhiTranslate]="userAddAction.translationKey"` doesn't really tell what the translation will look like, could we rather name it `label` instead?